integration-tests/maven/src/test/java/io/quarkus/maven/it/DevMojoIT.java

```diff
@@ -453,25 +452,20 @@ public void testThatNewBeanAreDiscovered() throws IOException, MavenInvocationEx
     }
 
     @Test
-    public void testErrorMessageWhenNoTarget() throws IOException, MavenInvocationException {
+    public void testNoErrorMessageWhenNoTarget() throws IOException, MavenInvocationException {
```

Comment:
Is this test still needed with auto-compile?

Reply:
probably because it's testing missing folders at two distinct levels.  one could probably make an argument for removing it but the extra coverage is probably worth it.